### PR TITLE
Prepare for state reconciliation & vpp restart

### PR DESCRIPTION
**Pull request by sknat**

### CODE REVIEW

File: cmd/calico_vpp_dataplane.go [hunk 1]
```diff
@@ -16,32 +16,26 @@
...
-	for i := 0; i < 20; i++ {
-		dat, err := ioutil.ReadFile(config.VppManagerStatusFile)
-		if err == nil && strings.TrimSpace(string(dat[:])) == "1" {
-			return nil
-		}
-		time.Sleep(1 * time.Second)
-	}
-	return errors.Errorf("Vpp manager not ready after 20 tries")
-}
+// var (
```
Comment (AloysAugustin) [inline cmd/calico_vpp_dataplane.go]: Could you remove this entirely?

**Review by AloysAugustin — COMMENTED** [review]
Globally lgtm :)

File: cmd/calico_vpp_dataplane.go (hunk 1 above)
Comment (sknat) [reply cmd/calico_vpp_dataplane.go]: Oh, indeed

Comment (sknat) [reply cmd/calico_vpp_dataplane.go]: Done

**Pull request merged**